TracksDemo/TracksDemo.xcodeproj/project.pbxproj

```diff
@@ -385,21 +389,21 @@
 			buildActionMask = 2147483647;
 			files = (
 			);
-			inputFileListPaths = (
-			);
 			inputPaths = (
 				"${PODS_ROOT}/Target Support Files/Pods-TracksDemo Mac/Pods-TracksDemo Mac-frameworks.sh",
 				"${BUILT_PRODUCTS_DIR}/Automattic-Tracks-iOS-macOS/AutomatticTracks.framework",
 				"${BUILT_PRODUCTS_DIR}/CocoaLumberjack-macOS/CocoaLumberjack.framework",
 				"${BUILT_PRODUCTS_DIR}/Reachability-macOS/Reachability.framework",
+				"${BUILT_PRODUCTS_DIR}/Sentry-macOS/Sentry.framework",
+				"${BUILT_PRODUCTS_DIR}/Sodium-macOS/Sodium.framework",
 			);
```

Comment:
Side note and future improvement reminder. Starting 1.10.0, CocoaPods uses the Xcode project `compatibilityVersion` value to decide whether to use a file with the list of frameworks or enumerate them in the project. See [this PR](https://github.com/CocoaPods/CocoaPods/commit/d4e8141261e40e3e2d9a201d627457a2c11eb545) and [this SO question](https://stackoverflow.com/questions/64254389/cocoapods-1-10-beta-2-unable-to-load-contents-of-file-list-xcfilelist).

Using the file list is better because it replaces this kind of noisy diff with a change in a dedicated file. To support it, we shall upgrade the project file to the Xcode 11.0 compatible version.

![image](https://user-images.githubusercontent.com/1218433/104245314-21a35500-54b8-11eb-8007-b1132daa0f47.png)

Reply:
I agree that the file list would be super nice!!! This one I think is a great fit for a future PR.